mms/go/cmd/gocqlxapp/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"goapp/internal/log"
+	"goapp/internal/scylla"
+
+	"github.com/gocql/gocql"
+	"github.com/scylladb/gocqlx"
+	"github.com/scylladb/gocqlx/qb"
+	"github.com/scylladb/gocqlx/table"
+	"go.uber.org/zap"
+)
+
+var (
```

Comment:
I'd suggest to delete this group and use :=

Reply:
They are contained in a separate little helper struct now.